ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/PartialConfig.hs

```diff
@@ -79,5 +82,8 @@ class ( UpdateLedger blk
 newtype WrapPartialLedgerConfig    blk = WrapPartialLedgerConfig    { unwrapPartialLedgerConfig    :: PartialLedgerConfig                   blk  }
 newtype WrapPartialConsensusConfig blk = WrapPartialConsensusConfig { unwrapPartialConsensusConfig :: PartialConsensusConfig (BlockProtocol blk) }
 
+deriving newtype instance (Typeable blk, ToCBOR   (PartialLedgerConfig blk))  => ToCBOR   (WrapPartialLedgerConfig    blk)
```

Comment:
Please remove the extra spaces before the `=>`s and before the last occurrences of `blk`.